docs/reference/announcements-release-notes/870/870-announcements.md

```diff
@@ -1,30 +1,38 @@
 ---
-id: announcements-870
-title: "8.7 Announcements"
+id: 870-announcements
+title: "Announcements"
 description: "Important changes and updates for the Camunda 8.7 release including deprecation & removal notices."
 ---
 
-import DeployDiagramImg from '../img/deploy-diagram-modal.png';
+import DeployDiagramImg from '../../img/deploy-diagram-modal.png';
 
 Important changes and updates for the Camunda 8.7 release are summarized below.
 
-| Scheduled release date | Scheduled end of maintenance | Release notes                                        | Blog                                                                                            |
-| :--------------------- | :--------------------------- | :--------------------------------------------------- | :---------------------------------------------------------------------------------------------- |
-| 8 April 2025           | 13 October 2026              | [8.7 release notes](/reference/release-notes/870.md) | [Announcing Camunda 8.7](https://camunda.com/blog/2024/11/camunda-8-7-releasing-february-2025/) |
+| Scheduled release date | Scheduled end of maintenance | Release notes                                                                        | Blog                                                                                            |
+| :--------------------- | :--------------------------- | :----------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------- |
+| 8 April 2025           | 13 October 2026              | [8.7 release notes](/reference/announcements-release-notes/870/870-release-notes.md) | [Announcing Camunda 8.7](https://camunda.com/blog/2024/11/camunda-8-7-releasing-february-2025/) |
 
 <!--- [Ad-hoc subprocesses](#)
 - [Document handling](#)
 - [RPA](#)
   - [Fetch RPA resource API](#)
   - [deployResourceAPI for RPA](#) -->
 
-- [Amazon Web Services (AWS) region support](#amazon-web-services-aws-region-support-saas)
-- [Web Modeler](#web-modeler)
-  - [Web Modeler Self-Managed deploy diagram change](#web-modeler-deploy-diagram-change)
+- [New features](#new-features)
+  - [Amazon Web Services (AWS) region support SaaS](#amazon-web-services-aws-region-support-saas)
+  - [Deploy diagram change Self-Managed](#web-modeler-deploy-diagram-change)
```

Comment:
There's a few instances where the SaaS/SM tags have slipped in to the title content here.